spec/system/candidate_interface/submitting/candidate_submits_application_with_an_invalid_course_option_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'rails_helper'
```

Comment:
Naming suggestion for filename and feature description—something along the lines of 'Candidate submits the application with a removed site'.